main.go

```diff
@@ -67,7 +67,12 @@ func bootstrap(c config.Configurator) {
 		log.Fatalf("Error writing bootstrap recording rules: %s", err)
 	}
 
-	err = prom.AppendRuleFile("/etc/config/bomb-squad/rules.yaml", c)
+	cfg, err := prom.AppendRuleFile("/etc/config/bomb-squad/rules.yaml", c)
+	if err != nil {
+		log.Fatalf("Error adding bootstrap recording rules to Prometheus config: %s", err)
+	}
+
+	err = config.WritePromConfig(cfg, c)
```

Comment:
This doesn't look like a unit test. Some other bugfix?

Reply:
Ah, this. Less bugfix, more "less side effects". It was annoying to test `prom.AppendRuleFile` when it was effectively an entirely impure function - it would read the ConfigMap, modify, and write it back, without giving me a chance to check the results in the tests (fake ConfigMap don't really update). So this was to move the side effect out of the tested function and into `main()`.